src/components/scrollbar-container/index.test.tsx

```diff
@@ -50,4 +63,12 @@ describe('ScrollbarContainer', () => {
     wrapper.setState({ showPanel: false });
     expect(wrapper.find('.scrollbar-container__panel')).toHaveLength(0);
   });
+
+  it('resets scroll position when new children are provided', () => {
```

Comment:
Is this the general scroll component? I'm not sure we want ALL things in a scroll component to reset its scroll position.

Reply:
The only other place it's used is the "channels list", in which case this will get triggered only when we add a new channel. So i had put it here, in general component. Is that..not correct? 🤔

Reply:
I was actually thinking about this a bit more. Will this impact the whole conversation list? If it does then it seems like there might be some weird user experiences if you've scrolled down to look at a conversation and then someone starts a new conversation with you...the children would change and your list would scroll to the top.

Reply:
Ok, i got it working from the `conversation-list-panel` component. So basically whenever a search input changes, and we're displays the "results", only then we scroll to top. But i had to use `refs` for this (like i had to define a new one).